--- epdfview-0.1.8/intl/config.charset~	2011-05-28 12:25:11.000000000 +0200
+++ epdfview-0.1.8/intl/config.charset	2011-08-16 11:39:42.838646280 +0200
@@ -133,7 +133,7 @@
 	         es_HN es_MX es_PA es_PE es_PY es_SV es_US es_UY es_VE et \
 	         et_EE eu eu_ES fi fi_FI fo fo_FO fr fr_BE fr_CA fr_CH fr_FR \
 	         fr_LU ga ga_IE gl gl_ES id id_ID in in_ID is is_IS it it_CH \
-	         it_IT kl kl_GL nl nl_BE nl_NL no no_NO pt pt_BR pt_PT sv \
+	         it_IT kl kl_GL nl nl_BE no no_NO pt pt_BR sv \
 	         sv_FI sv_SE; do
 	  echo "$l ISO-8859-1"
 	  echo "$l.iso-8859-1 ISO-8859-1"
@@ -181,7 +181,7 @@
 	  echo "$l.utf-8 UTF-8"
 	  echo "$l.utf-8@euro UTF-8"
 	done
-	for l in he he_IL iw iw_IL; do
+	for l in he iw iw_IL; do
 	  echo "$l ISO-8859-8"
 	  echo "$l.iso-8859-8 ISO-8859-8"
 	  echo "$l.cp-862 CP862"
@@ -358,7 +358,7 @@
 	done
 	for l in da_DK de_AT de_CH de_DE en_AU en_CA en_GB en_US es_ES \
 	         fi_FI fr_BE fr_CA fr_CH fr_FR is_IS it_CH it_IT la_LN \
-	         lt_LN nl_BE nl_NL no_NO pt_PT sv_SE; do
+	         lt_LN nl_BE nl no_NO pt sv_SE; do
 	  echo "$l.ISO_8859-1 ISO-8859-1"
 	  echo "$l.DIS_8859-15 ISO-8859-15"
 	done
@@ -406,7 +406,7 @@
 	done
 	for l in da_DK de_AT de_CH de_DE en_AU en_CA en_GB en_US es_ES \
 	         fi_FI fr_BE fr_CA fr_CH fr_FR is_IS it_CH it_IT nl_BE \
-	         nl_NL no_NO pt_PT sv_SE; do
+	         nl no_NO pt sv_SE; do
 	  echo "$l ISO-8859-1"
 	  echo "$l.ISO8859-1 ISO-8859-1"
 	  echo "$l.ISO8859-15 ISO-8859-15"
@@ -552,14 +552,14 @@
 	echo "nb_NO CP865" # not CP850 ??
 	echo "nl CP850"
 	echo "nl_BE CP850"
-	echo "nl_NL CP850"
+	echo "nl CP850"
 	echo "nn CP865"    # not CP850 ??
 	echo "nn_NO CP865" # not CP850 ??
 	echo "no CP865"    # not CP850 ??
 	echo "no_NO CP865" # not CP850 ??
 	echo "pt CP850"
 	echo "pt_BR CP850"
-	echo "pt_PT CP850"
+	echo "pt CP850"
 	echo "sv CP850"
 	echo "sv_SE CP850"
 	# ISO-8859-2 languages
@@ -615,7 +615,6 @@
 	echo "el_GR CP869"
 	# ISO-8859-8 languages
 	echo "he CP862"
-	echo "he_IL CP862"
 	# ISO-8859-9 languages
 	echo "tr CP857"
 	echo "tr_TR CP857"
--- epdfview-0.1.8/intl/localename.c~	2011-05-28 12:25:11.000000000 +0200
+++ epdfview-0.1.8/intl/localename.c	2011-08-16 11:40:40.585302218 +0200
@@ -1182,7 +1182,7 @@
       case LANG_DUTCH:
 	switch (sub)
 	  {
-	  case SUBLANG_DUTCH: return "nl_NL";
+	  case SUBLANG_DUTCH: return "nl";
 	  case SUBLANG_DUTCH_BELGIAN: /* FLEMISH, VLAAMS */ return "nl_BE";
 	  }
 	return "nl";
@@ -1269,7 +1269,7 @@
 	/* FIXME: Do they mean Hawaiian ("haw_US", 1000 speakers)
 	   or Hawaii Creole English ("cpe_US", 600000 speakers)?  */
 	return "cpe_US";
-      case LANG_HEBREW: return "he_IL";
+      case LANG_HEBREW: return "he";
       case LANG_HINDI: return "hi_IN";
       case LANG_HUNGARIAN: return "hu_HU";
       case LANG_IBIBIO: return "nic_NG";
@@ -1347,7 +1347,7 @@
       case LANG_PORTUGUESE:
 	switch (sub)
 	  {
-	  case SUBLANG_PORTUGUESE: return "pt_PT";
+	  case SUBLANG_PORTUGUESE: return "pt";
 	  /* Hmm. SUBLANG_PORTUGUESE_BRAZILIAN == SUBLANG_DEFAULT.
 	     Same phenomenon as SUBLANG_ENGLISH_US == SUBLANG_DEFAULT. */
 	  case SUBLANG_PORTUGUESE_BRAZILIAN: return "pt_BR";
--- epdfview-0.1.8/po/LINGUAS~	2011-05-28 12:25:14.000000000 +0200
+++ epdfview-0.1.8/po/LINGUAS	2011-08-16 11:41:13.648638135 +0200
@@ -1,2 +1,2 @@
 # Set of available languages.
-ar ca cs de el es eu fr gl hu he_IL it ja nl_NL pl pt_BR pt_PT ru sv vi zh_CN zh_TW
+ar ca cs de el es eu fr gl hu he it ja nl pl pt_BR pt ru sv vi zh_CN zh_TW
